m4_sessions/1706-inning/group_four.md

```diff
@@ -2,8 +2,62 @@
 
 ### Session Title:
 
+"Getting involved and giving back as a beginner"  
+
 ### Session Outcomes: 
 
+1. Doc to add to their portfolio
+2. Interest/list of OSS to make first PR
+3. Check list to work through for first PR (fulfills PD deliverables, including [blog post (all mods), PR (mod 4)])
+
+Feelings and thoughts:
+
+- people feel confident in their ability to contribute to the community
+- people have the knowledge of where to go to contribute
+- the knowledge of HOW to contribute (like OSS PR)
+
+
+
+
+Ideas:
+
+- videos
+- list of projects to contribute to
+- how to make an impact at Turing
+  - make PR on lessons
+  - make oss contributions
+  - PR checklist/template
+
 ### Session Outline:
 
+
+- do we have a responsibility to "give back" to the community?
+
+- what does "giving back" look like?
+
+High level, how helpful this can be
+
+- ever been helped by a blog?
+- ever been helped by another student in your cohort or a cohort ahead?
+- helped by a teacher?
+- a mentor?
+
+We'll talk about how to do all that. 
+
+- OSS contributions (a guide?)
+- Write down skills you already have to build confidence
+- Find list of places in turing repo where you could find, fix, submit PR w/fix.
+
+Walk through example of PR against Turing repo to fix problem
+
+Walk through example of "student interest group", pairing, be a mentor
+
+Go outside the bubble, volunteer in the community (try turing, etc)
```

Comment:
Could be cool to reach out to Rachel Warbelow and see what the upcoming Try Turing dates are and have people sign up in the session to be TAs.